packages/curve-ui-kit/src/features/market-position-details/BorrowInformation.tsx

```diff
@@ -65,25 +71,49 @@ export const BorrowInformation = ({
       <Metric
         size="medium"
         label={t`Borrow rate`}
-        value={borrowAPY?.value}
-        loading={borrowAPY?.value == null && borrowAPY?.loading}
+        value={borrowAPY?.rate}
+        loading={borrowAPY?.rate == null && borrowAPY?.loading}
         valueOptions={{ unit: 'percentage', color: 'warning', decimals: 2 }}
         notional={
-          borrowAPY?.thirtyDayAvgRate
+          borrowAPY?.averageRate
             ? {
-                value: borrowAPY.thirtyDayAvgRate,
+                value: borrowAPY.averageRate,
                 unit: { symbol: '% 30D Avg', position: 'suffix' },
                 decimals: 2,
               }
             : undefined
         }
+        valueTooltip={{
+          title: t`Borrow Rate`,
+          body: (
+            <MarketBorrowRateTooltip
+              marketType={marketType}
+              borrowRate={borrowAPY?.rate}
+              averageRate={borrowAPY?.averageRate}
+              rebasingYield={borrowAPY?.rebasingYield}
+              collateralSymbol={collateralValue?.collateral?.symbol}
+              periodLabel={borrowAPY?.averageRateLabel ?? ''}
+              extraRewards={borrowAPY?.extraRewards ?? []}
+            />
+          ),
+          placement: 'top',
+          arrow: false,
+          clickable: true,
+        }}
       />
       <Metric
         size="medium"
         label={t`Total debt`}
         value={totalDebt?.value}
         loading={totalDebt?.value == null && totalDebt?.loading}
         valueOptions={{ unit: { symbol: 'crvUSD', position: 'suffix' } }}
+        valueTooltip={{
+          title: t`Total Debt`,
+          body: <TotalDebtTooltip />,
```

Comment:
If the component is not rendering `<Tooltip>` the name should be clear that it's only the tooltip content
```suggestion
          body: <TotalDebtTooltipContent />,
```